main.py

```diff
@@ -1,4 +1,5 @@
 from ShapeDetector import paint
+# @Copyright 2021 joeyshi.tech
```

Comment:
Please consider changing this to a copyright for 102peterlee.me

Reply:
PR is already growing too large, left as a todo here https://github.com/102plee/AI_Projects/issues/2

Reply:
I'm afraid this PR can't be merged without addressing this pressing issue